### ISSUE TITLE
Colab error and how many recycles possible?

### ISSUE THREAD

**Issue by avilella** [dptech-corp/Uni-Fold#107]
Hi, I got this error from attempting to run the colab:

![image](https://user-images.githubusercontent.com/158007/224325548-a9527da4-a525-4a0f-90a7-aec95694e9bd.png)

I also wanted to ask how many recycles would be possible to run with two sequences, one 150aa long and the other 250aa long before running out of memory in a Colab (not the pro one).

Thanks

**Comment by ZiyaoLi** [comment]
would you please show me the outputs in the first block of the notebook, to check if Unicore is appropriately installed? Seems to me that the installation of unicore is failed. For recycling there is no upper bound, because it doesn’t cost more memory but more time.

**Comment by ZiyaoLi** [comment]
thx for the feedback.

colab (again) updated their python version from 3.8 to 3.9.16, which made the installation failed.

Will PR a fix, but for quicker you may change the `cp38`s in the unicore url to `cp39`s.

**Comment by avilella** [comment]
There were 4 occurrences of the token 'cp38', which I changed to 'cp39',
then started a multimer execution with 3 proteins, max_recycling_iters: 9,
num_ensembles: 2 , manual_seed: 42, times: 3.

It started the prediction step, so I think it's working now.

**Comment by ZiyaoLi** [comment]
should be fixed in #109 .

**Comment by burunduk57** [comment]
I just checked, and it works as is, without any editing. Thank you very much!

**Comment by busrasavas** [comment]
Hi, I got the same error about the unicore installation. Checked the unicore url, it seems updated. I attached the outputs of the related cells, I'd appreciate any help. Thanks in advance! 
![Screenshot 2023-05-08 at 01 57 00](https://user-images.githubusercontent.com/62547137/236706831-9f9f26b3-cd65-4436-9a0a-6fbf823dda2f.png)
![Screenshot 2023-05-08 at 01 56 23](https://user-images.githubusercontent.com/62547137/236706832-04cc7de7-986b-41e3-9954-4b30ab54a3b9.png)
![Screenshot 2023-05-08 at 01 55 05](https://user-images.githubusercontent.com/62547137/236706834-c096aba6-d0cf-426c-8e29-2530c10cd2d6.png)
